examples/spiders/manual_recaptcha.py

```diff
@@ -62,4 +61,4 @@ def make_screenshot(self, response: PuppeteerScreenshotResponse, **kwargs):
 
     @staticmethod
     def error(failure: Failure):
-        print(f"We are in error function!")
+        print("We are in error function!")
```

Comment:
Do we need this print?

Reply:
It is the way to explicitly show the user that he has failed the request.

Reply:
Then maybe better to use logging for such messages